Index: SPIRV/CMakeLists.txt
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- SPIRV/CMakeLists.txt	(revision 3ee5f2f1d3316e228916788b300d786bb574d337)
+++ SPIRV/CMakeLists.txt	(date 1596542072020)
@@ -92,14 +92,15 @@
 endif()
 
 if(ENABLE_OPT)
-    target_include_directories(SPIRV
-        PRIVATE ${spirv-tools_SOURCE_DIR}/include
-        PRIVATE ${spirv-tools_SOURCE_DIR}/source
-    )
-    target_link_libraries(SPIRV PRIVATE SPIRV-Tools-opt)
-    target_include_directories(SPIRV PUBLIC
-        $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/../External>
-        $<INSTALL_INTERFACE:${CMAKE_INSTALL_INCLUDEDIR}/External>)
+#    target_include_directories(SPIRV
+#        PRIVATE ${spirv-tools_SOURCE_DIR}/include
+#        PRIVATE ${spirv-tools_SOURCE_DIR}/source
+#    )
+#    target_link_libraries(SPIRV PRIVATE SPIRV-Tools-opt)
+#    target_include_directories(SPIRV PUBLIC
+#        $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/../External>
+#        $<INSTALL_INTERFACE:${CMAKE_INSTALL_INCLUDEDIR}/External>)
+    target_link_libraries(SPIRV PRIVATE ${CONAN_LIBS})
 endif(ENABLE_OPT)
 
 if(WIN32)
Index: CMakeLists.txt
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- CMakeLists.txt	(revision 3ee5f2f1d3316e228916788b300d786bb574d337)
+++ CMakeLists.txt	(date 1596542102470)
@@ -318,9 +318,9 @@
     add_subdirectory(External)
 endif()
 
-if(NOT TARGET SPIRV-Tools-opt)
-    set(ENABLE_OPT OFF)
-endif()
+#if(NOT TARGET SPIRV-Tools-opt)
+#    set(ENABLE_OPT OFF)
+#endif()
 
 if(ENABLE_OPT)
     message(STATUS "optimizer enabled")
